src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.application.name=LearningManagementSystem
```

Comment:
`application.properties` is fine, but could you please switch to yaml format (just to see the difference later :) )